package.json

```diff
@@ -13,7 +13,7 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
-    "openapi:bundle": "bundle-api-spec -i openapi/index.yaml.template -o openapi/index.yaml -V $npm_package_version",
+    "openapi:bundle": "bundle-api-spec -i openapi/index.template.yaml -o openapi/index.yaml -V $npm_package_version",
```

Comment:
This rename is ok, BUT be careful! In some other places we run something like `blablacommand *.yaml.template` so renaming the templates there could be a problem because you feed also the results to the blablacommand.

Reply:
[76222ad](https://github.com/pagopa/io-functions-service-messages/pull/47/commits/76222ad4aee01628403426d2f6349adb6509b5c1)